pkg/build/nodeimage/const_cni.go

```diff
@@ -20,11 +20,12 @@ package nodeimage
 The default CNI manifest and images are our own tiny kindnet
 */
 
-var defaultCNIImages = []string{"kindest/kindnetd:v20210119-d5ef916d"}
+var defaultCNIImages = []string{"aojea/kindnetd:fast"}
 
 const defaultCNIManifest = `
 # kindnetd networking manifest
 # would you kindly template this file
```

Comment:
TODO: we should migrate fully to patching in the future and do away with the template crud.
This will break support for older releases without the patching so we should wait some time.